webpack/assets/javascripts/react_app/components/Layout/Layout.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { VerticalNav } from 'patternfly-react';
+import { isEmpty } from 'lodash';
+import { noop } from '../../common/helpers';
+
+import TaxonomySwitcher from './components/TaxonomySwitcher';
+import UserDropdowns from './components/UserDropdowns';
```

Comment:
I've noticed that on click there is box around the dropdown, see 
![recorded](https://user-images.githubusercontent.com/20167/44095506-ad534ed6-9fe1-11e8-9c81-91086499df79.gif)

Reply:
fixed.